src/App.css

```diff
@@ -1,26 +0,0 @@
-@import '~antd/dist/antd.css';
```

Comment:
Where are you inserting the css file now?

Reply:
Are you still going to work with styled components? If you do, css files are not necessary.

Reply:
But in this casse we are using ant design library. They recommend to use using a plugin as you can see here: 
https://ant.design/docs/react/use-with-create-react-app

Can you do that? Like the recommended way.

Reply:
I really don't get why are you using a component library since we probably gonna need to wrap everything to get a desired behavior, but anyway if you wanna keep this anti lib you can keep it. I've just erase the css since it was not necessary to test our redux. When you want a css just create and import ant. It will gonna work just fine.

Reply:
Wee've disscussed that on the channel. If you think should be another way it would be awesome to start this discussion again on the ui/ux channel. Go ahead and ask there about it! 😸

Reply:
No problem, but just to be clear. I did'n delete anything about ant. I just removed the default css created by create-react-app

Reply:
Maybe you could add the css file on the index.css. Later, we can decide which approach we are going to work with!

Reply:
done!